docs/source/features/post_screening.rst

```diff
@@ -50,9 +50,12 @@ new record_id, based on the row number and starting at 0.
 Export Project
 --------------
 
-An ASReview project file can be downloaded containing all information needed to
-replicate the project or to import the project on a different device. To
-export your project:
+An ASReview project file can be exported from ASReview LAB. The file contains all data, decisions, and models. 
+replicate the project or to import the project on a different device. 
+The ASReview project file (.asreview) allows another person to continue your
+project exactly where you left off. 
```

Comment:
```suggestion
The ASReview project file (.asreview) can be exported from ASReview LAB. 
The file contains the dataset, review history, notes, and model configuration. 
It can be imported into ASReview LAB on a different device, which allows other
users to replicate the project, or continue the systematic review.
```